tensorflow/tools/ci_build/linux/mkl/basic-mkl-test.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
Instead of a new file, please feel free to modify and use this file as you need:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/tools/ci_build/linux/cpu/run_mkl.sh

Reply:
I want to fully align existing tests with what you run, so I would strongly prefer if we used the same script.

Reply:
@gunan Done.